pom.xml

```diff
@@ -38,14 +38,14 @@
                                 <include>**/src/**/*</include>
                                 <include>**/res/**/*</include>
                                 <include>**/lib/**/*</include>
-                                <include>**/bundle.json</include>
+                                <include>**/bundle.js</include>
```

Comment:
Both .js and .json are required for backwards compatibility with bundles still defined declaratively.